### DISCUSSION TITLE
[Bug]: Menu problem with template Massively in reduced size (phone) mode.

### DISCUSSION THREAD

**Discussion by founfabug** [GetPublii/Publii#1696] [Bug Report] [answered]
### Operating system

Win 10

### Publii version

0.46.1 build 16918

### Post editor

None

### Bug description

**Behaviour**: With "Massively" theme, if site has no main menu configured and site is displayed with restricted dimensions (phone screen) the hamburger menu appears. When hamburger menu opened, there is a large empty box displayed with no entries.
**Anticipated behaviour**: No hamburger menu displayed if no main menu exists.

### Steps to reproduce

**Reproduce**: Install an empty site, configure "Massively" as theme, preview site on browser with small dimensions (e.g. phone), observe hamburger menu, select hamburger menu, observe empty box.

### Relevant log output

```shell
N/A
```

**Comment by bobmitro** [comment]
Not related to the app.

**Comment by founfabug** [comment]
My apologies if this is the incorrect place for reporting this bug. The Publii marketplace indicates "this theme has been expertly ported to Publii CMS by the Publii Team." from this I assumed the behavior of the theme in response to Publii menu items should be reported here. If you would please direct me to the correct location I will follow through as appropriate. Thank you.

**Comment by bobmitro** [comment, marked as answer]
I'll turn this into a discussion, and next time, please post any non-app-related issues there.
